includes/admin/views/html-admin-page-addons.php

```diff
@@ -30,7 +30,13 @@
 
 	<p><?php esc_html_e( 'This is an optional plugin that is seperately installed and activated. Its primary purpose is to ensure Classic Commerce compatibility with extensions that are dependent on the installation of WooCommerce.', 'classic-commerce' ); ?></p>
 	
-	<p><?php printf( __( '<strong>First, delete WooCommerce</strong> in order to run the Compatibility plugin. The two cannot co-exist!', 'classic-commerce' ) ); ?></p>
+	<p><?php printf( __( 'The compatibility plugin can be downloaded from GitHub by clicking <strong><a href="%s">here</a></strong>. Save to your hard drive.', 'classic-commerce' ), 'https://github.com/Classic-Commerce/cc-compat-woo/releases/download/v9999.0/woocommerce.zip' ); ?></p>
```

Comment:
I have made the plugin release available as in the chat. Please use this link  in the plugin. https://classiccommerce.cc/docs/cc-compatibility-for-woo-addons-plugin/